### PR TITLE
[INFRA] Add known contributor translation

### PR DESCRIPTION
**Pull request by pan3793**
### _Why are the changes needed?_

This PR proposes to add a contributor name translation mapping, it's for the contributor's contributions calculation.

Note, that this PR only contains the mapping, but not the scripts (I still need to polish them).

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/master/contributing/code/testing.html#running-tests) locally before make a pull request

### _Was this patch authored or co-authored using generative AI tooling?_

No

### CODE REVIEW

File: build/release/known_translations [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# This is a mapping of names to be translated.
+# The format expected on each line should be: <GitHub ID> - <Full Name>
```
Comment (pan3793) [inline build/release/known_translations:2]: The mapping does not cover the following ids
```
biangjuang
weixi62961
FourSpaces
mrtisttt
XorSum
fantasticKe
camper42
mans2singh
touchida
CavemanIV
QianyongY
uicosp
rhh777
huangzhir
Kyofin
huapan123456
Karsonnel
wangyepeng2
yehere
phyyou
liaoyt
runzhliu
```

Comment (cxzl25) [reply build/release/known_translations:19]: add
```
XorSum
CavemanIV
QianyongY
mattshma
```

**Review by cxzl25 — APPROVED** [review]

**Comment by bowenliang123** [comment]
Are there `yaooqinn`, `pan3793` and `link3280` in the list?

**Comment by pan3793** [comment]
@bowenliang123 the names are extracted from the commits, we already use the full name in that.

**Comment by bowenliang123** [comment]
> @bowenliang123 the names are extracted from the commits, we already use the full name in that.

Thanks. Acknowledged.

**Comment by pan3793** [comment]
Thanks, merged to master/1.8

**Pull request closed without merging**